js/ccex.js

```diff
@@ -131,6 +133,48 @@ module.exports = class ccex extends Exchange {
         return this.parseOrderBook (orderbook, undefined, 'buy', 'sell', 'Rate', 'Quantity');
     }
 
+    async fetchOrderBooks(symbols = undefined, params = {}){
+        await this.loadMarkets ();
+        let orderBooks = [];
+        let orderBooksResult = [];
+
```

Comment:
Empty lines are not allowed inside methods by the following rules: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes (we really have to follow them all literally).

Reply:
i try to find any empty lines and to have a single line between methods.